pkgs/by-name/pu/pulsemeeter/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  libpulseaudio,
+  libappindicator,
+  gobject-introspection,
+  wrapGAppsHook3,
+  callPackage,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "pulsemeeter";
+  version = "1.2.14";
+
+  src = fetchFromGitHub {
+    owner = "theRealCarneiro";
+    repo = "pulsemeeter";
+    tag = "v${version}";
+    hash = "sha256-QTXVE5WvunsjLS8I1rgX34BW1mT1UY+cRxURwXiQp5A=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    pulsectl
+    pygobject3
+  ];
+
+  nativeBuildInputs = [
+    wrapGAppsHook3
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    libappindicator
+    libpulseaudio
```

```suggestion
    libpulseaudio
    bash
```

There's a `#!/usr/bin/env bash` shebang in `$out/bin/pmctl`. Add this to replace it.

You may need `patchShebangs $out/bin/pmctl` in `postInstall`. Not sure.